### PR TITLE
[12.0][IMP] account_payment_order: Add compatibility with partner_bank_active

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa TT26023

### CODE REVIEW

File: account_payment_order/models/account_move_line.py [hunk 1]
```diff
@@ -59,8 +59,14 @@ def _prepare_payment_line_vals(self, payment_order):
             # in this case
         if payment_order.payment_type == 'outbound':
             amount_currency *= -1
-        partner_bank_id = self.partner_bank_id.id or first(
-            self.partner_id.bank_ids).id
+        partner_bank_id = self.partner_bank_id.id
+        if not partner_bank_id:
+            bank_ids = self.partner_id.bank_ids
```
Comment (pedrobaeza) [inline account_payment_order/models/account_move_line.py:64]: But having the module, is this returning full list even with archived banks?

Comment (pedrobaeza) [reply account_payment_order/models/account_move_line.py:64]: Maybe there's a context `active_test=False` around?

Comment (pedrobaeza) [reply account_payment_order/models/account_move_line.py:64]: Seen.

**Review by pedrobaeza — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
Thanks! :+1:

**Review by victoralmau — APPROVED** [review]

**Pull request merged**